pkg/consts/file.go

```diff
@@ -18,6 +18,9 @@ const (
 	// NoLongerSharedDirID is the identifier of the directory where the files &
 	// folders removed from a sharing but still used via a reference are put
 	NoLongerSharedDirID = "io.cozy.files.no-longer-shared-dir"
+	// DrivesDirID is the identifier of the directory where the
+	// (shared|external) drives are saved.
+	DrivesDirID = "io.cozy.files.drives-dir"
```

Comment:
Shouldn't this be something like `io.cozy.files.shared-drives-dir` since its name is `Shared drives`?